designer/server/src/main/scala/pl/touk/nussknacker/ui/db/entity/StickyNotesEntityFactory.scala

```diff
@@ -0,0 +1,112 @@
+package pl.touk.nussknacker.ui.db.entity
+
+import pl.touk.nussknacker.engine.api.LayoutData
+import pl.touk.nussknacker.engine.api.process.{ProcessId, VersionId}
+import pl.touk.nussknacker.ui.api.description.stickynotes.StickyNoteEvent
+import pl.touk.nussknacker.ui.api.description.stickynotes.StickyNoteEvent.StickyNoteEvent
+import slick.lifted.{ProvenShape, TableQuery => LTableQuery}
+import slick.sql.SqlProfile.ColumnOption.NotNull
+import io.circe.syntax._
+import io.circe._
+import pl.touk.nussknacker.ui.api.description.stickynotes.Dtos.{
+  Dimensions,
+  StickyNote,
+  StickyNoteCorrelationId,
+  StickyNoteId
+}
+
+import java.sql.Timestamp
+import java.util.UUID
+
+trait StickyNotesEntityFactory extends BaseEntityFactory {
+
+  import profile.api._
+
+  val processVersionsTable: LTableQuery[ProcessVersionEntityFactory#ProcessVersionEntity]
+
+  class StickyNotesEntity(tag: Tag) extends Table[StickyNoteEventEntityData](tag, "sticky_notes") {
+
+    def id                = column[StickyNoteId]("id", O.PrimaryKey, O.AutoInc)
+    def noteCorrelationId = column[StickyNoteCorrelationId]("note_correlation_id", NotNull)
+    def content           = column[String]("content", NotNull)
+    def layoutData        = column[LayoutData]("layout_data", NotNull)
+    def color             = column[String]("color", NotNull)
+    def dimensions        = column[Dimensions]("dimensions", NotNull)
```

Comment:
`Dimensions` looks like property of `LayoutData`. Is it on purpose to keep it separated?

Reply:
Yes, `dimensions` is specific for stickyNotes and `LayoutData` is used among nodes as well